app/components/support_interface/reference_with_feedback_component.rb

```diff
@@ -8,32 +8,40 @@ class ReferenceWithFeedbackComponent < ViewComponent::Base
              :relationship,
              :feedback_status,
              :consent_to_be_contacted,
+             :relationship_confirmation,
+             :relationship_correction,
+             :referee_type,
```

Comment:
Not for this PR but wondering why support is able to edit the `reference` and `relationship to candidate` but not the `type of reference` and `relationship confirmed by referee`?

Reply:
Yeah, I wondered that too. I think maybe just because we haven't built that functionality yet? 🤷